Exercises/loops/loops.js

```diff
@@ -16,3 +30,27 @@ const arrayToFilter = Array.from(new Array(1000), () => {
 
   return null;
 });
+
+//const newArray = arrayToFilter.filter(item => Number.isInteger(item));
+
+let i = 0;
+
+while (i < arrayToFilter.length) {
+  if (!Number.isInteger(arrayToFilter[i])) {
+    arrayToFilter.splice(i, 1);
+    i--;
+  }
+  i++;
+}
```

Comment:
This can also be done without changing initial array in the loop but by creating a new array just like `.filter()` method does.